x/concentrated-liquidity/lp.go

```diff
@@ -1,6 +1,32 @@
 package concentrated_liquidity
 
-import sdk "github.com/cosmos/cosmos-sdk/types"
+import (
+	fmt "fmt"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+
+	types "github.com/osmosis-labs/osmosis/v12/x/concentrated-liquidity/types"
+)
+
+func (k Keeper) JoinPool(ctx sdk.Context, poolId uint64, owner sdk.AccAddress, tokenIn sdk.Int, lowerTick sdk.Int, upperTick sdk.Int) (numShares sdk.Int, err error) {
```

Comment:
Should we add some basic tests right away? Just making sure that tick / liquidity are updated etc

Reply:
I actually disagree with adding test at this point, considering that it doesn't have any logic or calculation besides updating state, which is not worth it considering time needed to do this. I would propose adding test on the next milestone (the second milestone)